pkg/cmd/server/origin/run_components.go

```diff
@@ -404,13 +405,21 @@ func (c *MasterConfig) RunServiceServingCertController(client *kclientset.Client
 // RunImageImportController starts the image import trigger controller process.
 func (c *MasterConfig) RunImageImportController() {
 	osclient := c.ImageImportControllerClient()
-	importRate := float32(c.Options.ImagePolicyConfig.MaxScheduledImageImportsPerMinute) / float32(time.Minute/time.Second)
-	importBurst := c.Options.ImagePolicyConfig.MaxScheduledImageImportsPerMinute * 2
+
+	var limiter flowcontrol.RateLimiter = nil
+	if c.Options.ImagePolicyConfig.MaxScheduledImageImportsPerMinute == -1 {
+		limiter = flowcontrol.NewTokenBucketRateLimiter(math.MaxFloat32, math.MaxInt32)
```

Comment:
I was looking for something like that and didn't see it.  I'll look again.  

Any negative number greater that -1 is rejected in `ValidateImagePolicyConfig` but I'll change this anyway